docs/user/library/jdbc/db2.rst

```diff
@@ -69,6 +69,25 @@ For more information check the java docs for:
 * DB2NGJNDIDataStoreFactory
 * DB2NGDataStoreFactory
 
+Advanced Geotools Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-------------------+------------------------------------------+
+| Parameter         | Description                              |
++===================+==========================================+
+| "loose bbox"      | Flag controlling loose bbox comparisons, |
+|                   | default is true                          |
++-------------------+------------------------------------------+
+| "use selectivity" | Flag to raise the priority of spatial    |
+|                   | indices for the db2 optimizer,           |
+|                   | default is false                         |
++-------------------+------------------------------------------+
```

Comment:
HI Jody

This correct, SELECTIVITY 0.0001 is a hint for the optimizer, but the
default is false for backward compatibility.

Cheers
Christian

On Tue, Aug 18, 2015 at 6:24 AM, Jody Garnett <notifications@github.com>
wrote:

> In docs/user/library/jdbc/db2.rst
> <https://github.com/geotools/geotools/pull/929#discussion_r37263195>:
>
> > @@ -69,6 +69,25 @@ For more information check the java docs for:
> >  * DB2NGJNDIDataStoreFactory
> >  * DB2NGDataStoreFactory
> >
> > +Advanced Geotools Parameters
> > +^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> > +
> > ++-------------------+------------------------------------------+
> > +| Parameter         | Description                              |
> > ++===================+==========================================+
> > +| "loose bbox"      | Flag controlling loose bbox comparisons, |
> > +|                   | default is true                          |
> > ++-------------------+------------------------------------------+
> > +| "use selectivity" | Flag to raise the priority of spatial    |
> > +|                   | indices for the db2 optimizer,           |
> > +|                   | default is false                         |
> > ++-------------------+------------------------------------------+
>
> It does help. If SELECTIVITY means something to DB2 users (which it looks
> like it does since you have it in an SQL example) then I am happy to keep
> the word.
>
> The description though is about an effect, SELECTIVITY seems to be a
> detail on how it is implemented?
>
> How about: SELECTIVITY: Value of true will inject SELECTIVITY 0.0001 hint
> to the optimizer to raise the priority of the spatial index. Default is
> false.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geotools/geotools/pull/929/files#r37263195>.
>



-- 
DI Christian Mueller MSc (GIS), MSc (IT-Security)
OSS Open Source Solutions GmbH